helper/schema/resource_test.go

```diff
@@ -44,7 +45,7 @@ func TestResourceApply_create(t *testing.T) {
 		},
 	}
 
-	actual, err := r.Apply(s, d, nil)
+	actual, err := r.Contextify().Apply(context.Background(), s, d, nil)
```

Comment:
throughout this file I was "lazy" and instead of rewriting all the CRUD func assignments to the context aware version I just invoke `Contextify()` on each resource before calling `Apply/Refresh/etc`.